### ISSUE TITLE
[ToggleSplitButton] Inconsistent Disabled colors and poor contrast 

### ISSUE THREAD

**Issue by niels9001** [microsoft/microsoft-ui-xaml#6500]
### Describe the bug

When is ToggleSplitButton is IsChecked and disabled, the Content gets very hard to read. (See screenshots below). It's also incosistent when comapring it to the a disabled Button (with AccentButtonStyle) or disabled ToggleButton.

### Steps to reproduce the bug

Add

`<Grid>
<muxc:ToggleSplitButton IsEnabled="False" IsChecked="True" Margin="12" Content="ToggleSplitButton with checked state and disabled" />
                <Button Style="{ThemeResource AccentButtonStyle}" Content="Button with AccentButtonStyle and disabled" IsEnabled="False" Margin="12" />
                <ToggleButton IsChecked="True" Content="ToggleButton and disabled" IsEnabled="False" Margin="12" />
</Grid>`

### Expected behavior

- Better contrast meeting accessibility requirements
- Similar visual to AccentButtonStyle and ToggleButton

### Screenshots

Light theme

![image](https://user-images.githubusercontent.com/9866362/146681761-276eb1ef-8d89-4d43-89b6-d80d84f65e83.png)

Dark theme

![image](https://user-images.githubusercontent.com/9866362/146681867-75830912-6a58-4979-990c-e91290745dcf.png)

### Windows app type

- [X] UWP
- [X] Win32

### Device form factor

Desktop

**Comment by robloo** [comment]
Related to #6469

**Comment by robloo** [comment]
This one is easier to fix and should just require inverting the colors in the resource brushes.

**Comment by StephenLPeters** [comment]
@chigy FYI, can you suggest the proper behavior?

**Comment by Jay-o-Way** [comment]
> This one is easier to fix and should just require inverting the colors in the resource brushes.

So, what's keeping this back? :) @robloo

**Comment by robloo** [comment]
1. Microsoft design team has to decide things like this. They never replied.
2. I've moved away from WinUI so won't be looking at this.

**Comment by crutkas** [comment]
@chigy is this something being Looked at

**Comment by niels9001** [comment]
> @chigy is this something being Looked at

WinUI does not have Accent button styles beyond the default button. So this is a custom style we've created.

**Comment by DmitriyKomin** [comment]
@niels9001, thank you for reporting this.  Trying the repro XAML on WinAppSDK 1.5 / Windows 11, readability appears better than the previous (blue) screenshot:

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/45051803/e065388f-2964-4021-92d1-f634cc46c299)
